src/track/text-style.js

```diff
@@ -15,12 +15,27 @@
 class TextStyle {
 
   /**
+   * Defined set of font families
+   * @enum {Object.<string, string>}}
+   * @export
+   */
+  static FontFamily: {[string]: string} = {
+    "ARIAL": "Arial",
+    "ROBOTO": "Roboto",
```

Comment:
I wouldn't expose Roboto and other custom non-system fonts in the enum.
Maybe Hillel can assist with the font choice but I would go with:
'sans-serif' and 'serif'
other than that, the custom skin can define his own custom fonts and add it to the font options.